src/include/storage/table/table_heap.h

```diff
@@ -92,7 +93,17 @@ class TableHeap {
    */
   void UpdateTupleInPlaceUnsafe(const TupleMeta &meta, const Tuple &tuple, RID rid);
 
+  /** For binder tests */
+  static auto CreateEmptyHeap(bool create_table_heap = false) -> std::unique_ptr<TableHeap> {
+    // The input parameter should be false in order to generate a empty heap
+    assert(!create_table_heap);
+    return std::unique_ptr<TableHeap>(new TableHeap(create_table_heap));
```

Comment:
maybe we prefer to use make_unique.

Reply:
Actually I've tried `std::make_unique` before, but seems it can't be apply to private constructor, due to instantiations of the inner templates.

Reply:
Yeah, my mistake. https://stackoverflow.com/questions/65286678/make-unique-cannot-access-private-constructor-in-static-member

Reply:
Okay that's interesting... 🤪